crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4855,3 +4936,97 @@ impl From<AdyenStatus> for storage_enums::PayoutStatus {
         }
     }
 }
+
+impl<'a> TryFrom<(&domain::NetworkTokenData, Option<Secret<String>>)> for AdyenPaymentMethod<'a> {
```

Comment:
Avoid using primitives and strings alongside structs in `From` / `TryFrom` implementations, it's easy to forget what value the string should be, so an incorrect value may be sent instead. You can take up this refactor in a separate PR.